### ISSUE TITLE
How to edit/change RGB values of video?

### ISSUE THREAD

**Issue by bathnsahin** [scikit-video/scikit-video#75]
hello, i want to change RGB values of my video. How can I do this?

**Comment by beyondmetis** [comment]
This question seems very general. Can you be more specific? Once a video is loaded, you can edit values directly in python.

**Comment by bathnsahin** [comment]
Yes, I can :) I am trying to code a background algorithm which reflects by pixel moving. I tried to Backgroundsubtractor but I couldn't take the right result for me. I can see RGB values of my all frames. But I want to change my pixel RGB values when there is no pixel moving.  @beyondmetis

**Comment by beyondmetis** [comment]
If I understand you correctly, you are either trying to design a background subtraction algorithm, or you are trying to visualize the results of one. You can always mask your original video, then save the result as output. Is this what you want to do?

**Comment by bathnsahin** [comment]
Unfortunatelly not. My aim is to define contour with my background.

**Comment by Meixu-Chen** [comment]
scikit-video reads and writes videos as NumPy arrays, so editing pixel values is just standard NumPy array manipulation. For example: `videodata = skvideo.io.vread('video.mp4')` gives you a `(T, H, W, 3)` uint8 array — you can modify it with any NumPy operation, then write it back with `skvideo.io.vwrite('out.mp4', videodata)`. Closing as a usage question rather than a bug.
